regolith/helpers/a_manurevhelper.py

```diff
@@ -0,0 +1,125 @@
+"""Builder for Current and Pending Reports."""
+import datetime as dt
+import sys
+import time
+from argparse import RawTextHelpFormatter
+
+import nameparser
+
+from regolith.helpers.basehelper import SoutHelperBase, DbHelperBase
+from regolith.dates import month_to_int, month_to_str_int
+from regolith.fsclient import _id_key
+from regolith.sorters import position_key
+from regolith.tools import (
+    all_docs_from_collection,
+    filter_grants,
+    fuzzy_retrieval,
+)
+
+ALLOWED_TYPES = ["nsf", "doe", "other"]
+ALLOWED_STATI = ["invited", "accepted", "declined", "downloaded", "inprogress",
+                 "submitted", "cancelled"]
+
+
+def subparser(subpi):
+    subpi.add_argument("name", help="Full name of the first author",
+                       )
+    subpi.add_argument("due_date", help="due date in form YYYY-MM-DD in quotes", default=''
+                       )
+    subpi.add_argument("-c", "--recommendation",
+                       help="Recommendation for this manuscript: reject, smalledits, or majoredits", default=None
+                       )
+    subpi.add_argument("-d", "--database",
+                       help="The database that will be updated. Defaults to "
+                            "first database in the regolithrc.json file."
+                       )
+    subpi.add_argument("-r", "--reviewer",
+                       help="name of the reviewer.  Defaults to sbillinge"
+                       )
+    subpi.add_argument("-s", "--status",
+                       help=f"status, from {ALLOWED_STATI}. default is submitted"
+                       )
+    subpi.add_argument("-t", "--title",
```

Comment:
I think this should be required, not optional

Is there a "requester" field in the schema, as there is in proposal Review?  If so can we have an optional argument for that?  If not, please could you add it to the schema (and here?)

Same for "journal"

Reply:
There is no requester in refereeReports in schemas.py, but "journal" exists. I am adding requester.